packages/vue-components/src/annotations/AnnotatePoint.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <div>
+    <span
+      v-if="isMounted"
+      class="popover-annotation"
+      :x="x"
+      :y="y"
+      :style="pointPosition"
+    >
+      <portal v-if="targetEl.id" :to="'popover:' + targetEl.id">
+        <h3 v-if="hasHeader" class="popover-header">
+          <slot name="header"></slot>
+        </h3>
+        <div class="popover-body">
+          <slot name="content"></slot>
+        </div>
+      </portal>
+
+      <v-dropdown
```

Comment:
I just followed the implementation details from [the docs](https://floating-vue.starpad.dev/guide/component.html) page. From what I've seen, the implementation between v-dropdown and v-popover effectively perform the same way, so I can switch to either one.

Reply:
From what I interpreted from their docs, they use `VDropdown` because they [import `Dropdown` as `VDropdown`](https://floating-vue.starpad.dev/guide/installation.html#vue-2) (though in our case, we import `Dropdown` as `VPopover`, since they don't have a `VPopover` component). So I'm confused as to why `v-dropdown` works. O.o (correct me if I'm wrong?)

e.g., `import { VDropdown } from 'floating-vue';` throws an error since there's no such component, but `import { Dropdown } from 'floating-vue';` works since there is a `Dropdown` component.

That aside, I think it may be better to use `v-popover` to follow the Popover implementation? This ensures that both the MarkBind Popover component and the MarkBind Annotations component are using the same implementation for popovers, since I'm not sure where `VDropdown` is coming from.

Reply:
> e.g., `import { VDropdown } from 'floating-vue';` throws an error since there's no such component, but `import { Dropdown } from 'floating-vue';` works since there is a `Dropdown` component.

Honestly not sure why it works as well, but I did try just `dropdown` but it did not work so I switched to `v-dropdown` and it somehow started to work ;/.


> That aside, I think it may be better to use `v-popover` to follow the Popover implementation? This ensures that both the MarkBind Popover component and the MarkBind Annotations component are using the same implementation for popovers, since I'm not sure where `VDropdown` is coming from.

Ok sounds good, will change to v-popover instead then.